### DISCUSSION TITLE
How can I get Quickdraw badge?!

### DISCUSSION THREAD

**Discussion by daiaxexe** [community/community#20680] [Discussions] [answered]
I don't know how I can get Quickdraw badge? please let me know

**Comment by owerdogan** [comment, marked as answer]
Hey @daiaxexe .
First of all, I think it would be better to ask the question in the profile section. But to answer briefly, you can see the details and how to get all the achievements by looking at this article.

https://github.com/Schweinepriester/github-profile-achievements

Hope this will be help you.
